lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -504,8 +504,8 @@ function _dosomething_user_send_to_message_broker() {
   global $user;
   $account = $user;
   if (module_exists('dosomething_global')) {
-    $user_language = dosomething_global_get_user_language($account);
-    $user_country_code = dosomething_global_convert_language_to_country($user_language);
+    $user_language = dosomething_global_get_language($account);
+    $user_country_code = dosomething_global_get_country();
```

Comment:
Since this block is repeated in two places, is there any way we can abstract this into a helper function?